src/scenes/Search/store.js

```diff
@@ -1,93 +1,85 @@
-import _ from 'lodash';
-import { observable, runInAction, action, reaction } from 'mobx';
-import { Phases, OracleStatus, SortBy } from 'constants';
-import { searchOracles, searchTopics } from '../../network/graphql/queries';
+import { isEmpty, filter, debounce } from 'lodash';
+import { observable, action, reaction } from 'mobx';
+import { EVENT_STATUS, SortBy } from 'constants';
+import { searchEvents } from '../../network/graphql/queries';
+
+const INIT_VALUES = {
+  phrase: '',
+  loading: false,
+  tabIdx: 0,
+  events: [],
+  bets: [],
+  sets: [],
+  votes: [],
+  withdraws: [],
+};
 
-const TAB_BET = 0;
-const TAB_VOTE = 1;
-const TAB_SET = 2;
-const TAB_WITHDRAW = 3;
 export default class SearchStore {
-  @observable oracles = [];
-  @observable phrase = '';
-  @observable loading = false;
-  @observable loaded = false;
-  @observable tabIdx = 0;
-  @observable events = [];
-  @observable bets = [];
-  @observable votes = [];
-  @observable sets = [];
-  @observable withdraws = [];
+  @observable phrase = INIT_VALUES.phrase;
+  @observable loading = INIT_VALUES.loading;
+  @observable tabIdx = INIT_VALUES.tabIdx;
+  @observable events = INIT_VALUES.events;
+  @observable bets = INIT_VALUES.bets;
+  @observable sets = INIT_VALUES.sets;
+  @observable votes = INIT_VALUES.votes;
+  @observable withdraws = INIT_VALUES.withdraws;
 
   constructor(app) {
     this.app = app;
-    reaction( // whenever the locale changes, update locale in local storage and moment
-      () => this.phrase,
-      () => {
-        if (_.isEmpty(this.phrase)) {
-          this.loading = false;
-          this.loaded = false;
-          this.oracles = [];
-          this.withdraws = [];
-          this.events = [];
-          this.bets = [];
-          this.votes = [];
-          this.sets = [];
-        }
-      },
-    );
+
     reaction(
-      () => this.app.global.syncBlockNum + this.app.global.online,
+      () => this.app.global.online,
```

Comment:
previously fetch on new block in case the search result changes event status.

now that new block comes out too quickly, guess we can remove it for now.